api/ocsp/ocspsign.go

```diff
@@ -34,10 +36,11 @@ func NewHandler(s ocsp.Signer) http.Handler {
 
 // This type is meant to be unmarshalled from JSON
 type jsonSignRequest struct {
-	Certificate string `json:"certificate"`
-	Status      string `json:"status"`
-	Reason      int    `json:"reason,omitempty"`
-	RevokedAt   string `json:"revoked_at,omitempty"`
+	Certificate string      `json:"certificate"`
+	Status      string      `json:"status"`
+	Reason      int         `json:"reason,omitempty"`
+	RevokedAt   string      `json:"revoked_at,omitempty"`
+	IssuerHash  crypto.Hash `json:"issuer_hash,omitempty"`
```

Comment:
I think it would make more sense to have this as a string; otherwise interacting with the API would require some knowledge of the Go standard library.